app/res/xml/preferences_developer.xml

```diff
@@ -173,4 +173,10 @@
         android:entryValues="@array/pref_enabled_vals"
         android:key="cc-enable-certificate-transparency"
         android:title="Certificate Transparency"/>
+    <EditTextPreference
+        android:dialogTitle="Log Submission Server"
```

Comment:
nit: We should also rename it to have the same `Override` as title.

Reply:
And I forgot the word "Custom" too. Corrected [0069632](https://github.com/dimagi/commcare-android/pull/2852/commits/0069632157b68b90c94cf59b3d6d5acf4d9e34d6)

Reply:
Ahh I think if we use `Custom` , we should remove the `Override` i.e. use one of them instead of both.

Reply:
Agree